sdk/keyvault/Azure.Security.KeyVault.Keys/src/DeletedKey.cs

```diff
@@ -35,12 +35,12 @@ public Uri RecoveryId
         }
 
         /// <summary>
-        /// The time when the key was deleted, in UTC.
+        /// Gets a <see cref="DateTimeOffset"/> of when the key was deleted.
         /// </summary>
         public DateTimeOffset? DeletedOn { get; internal set; }
 
         /// <summary>
-        /// The time when the key is scheduled to be purged, in UTC
+        /// Gets a <see cref="DateTimeOffset"/> for when the deleted key will be purged.
```

Comment:
But users don't give it a `DateTimeOffset` in UTC. They give it a `DateTimeOffset` in whatever form they want, which is always correctly offset or equal to UTC. When we use `DateTimeOffset.ToUnixTimeSeconds()` it will always be in UTC becuase unixtime is always UTC. `DateTime` had issues that would've required users to be mindful of UTC because an offset was optional. It's not with `DateTimeOffset`.

The swagger only says this because it has to: it's just an integer representing unixtime. Because we use `DateTimeOffset`, it's an unnecessary burden to have users worry about converting since `DateTimeOffset` takes care of that for them which is, I presume, why we use it throughout Azure SDK.